### ISSUE TITLE
Support schema inspect for Mysql table with column "Timestamp/Datetime default on update"

### ISSUE THREAD

**Issue by mindbergh** [ariga/atlas#471]
We have a column that is in Datetime type and has the [default on update definition](https://dev.mysql.com/doc/refman/5.7/en/timestamp-initialization.html). For example:

```
CREATE TABLE `foo` (
 `id`			INT,
 `update_time`	DATETIME(6)  NOT NULL DEFAULT CURRENT_TIMESTAMP(6) ON UPDATE CURRENT_TIMESTAMP(6)

  PRIMARY KEY (`id`),
)
```

When I try to inspect the schema, I got the following error: 
```
atlas schema inspect -d "mysql://root:secret@tcp(localhost:12334)/my_database"
Error: mysql: unknown attribute "on update current_timestamp(6)"
```

Mysql version: 5.7.35

Atlas version: 
```
atlas CLI version v0.3.1
https://github.com/ariga/atlas/releases/tag/v0.3.1
```

**Comment by masseelch** [comment]
Hey @mindbergh, thank you for reaching out to us and reporting this. I will have a look at this :-)

**Comment by masseelch** [comment]
@mindbergh Please try the latest master and let me know if there still is an issue.

**Comment by mindbergh** [comment]
Thanks for the quick fix, it works now. I'm able to inspect the column for the default current_timestamp(6). But it still doesn't give the `on update current_timestamp(6)` attribute. I guess it will be addressed together in #472?

**Comment by masseelch** [comment]
> @masseelch Thanks for the quick fix, it works now. I'm able to inspect the column for the default current_timestamp(6). But it still doesn't give the `on update current_timestamp(6)` attribute. I guess it will be addressed together in #472?

@mindbergh An oversight on my sight. I'll add the field to the rendered HCL.

**Comment by masseelch** [comment]
@mindbergh I just added the `on_update` field to the column definition in HCL. Closing again, but please let me know if there are more issues.
